lib/AnyEvent/RabbitMQ/Channel.pm

```diff
@@ -543,13 +543,14 @@ sub consume {
 
     my $consumer_cb = delete $args{on_consume} || sub {};
     my $cancel_cb   = delete $args{on_cancel} || sub {};
+    my $no_ack      = delete $args{no_ack} || 1;
```

Comment:
You are correct, good catch.

On Sat, Mar 21, 2015 at 12:33 PM, davel <notifications@github.com> wrote:

> In lib/AnyEvent/RabbitMQ/Channel.pm
> <https://github.com/bobtfish/AnyEvent-RabbitMQ/pull/15#discussion_r26893983>
> :
>
> > @@ -543,13 +543,14 @@ sub consume {
> >
> >      my $consumer_cb = delete $args{on_consume} || sub {};
> >      my $cancel_cb   = delete $args{on_cancel} || sub {};
> > +    my $no_ack      = delete $args{no_ack} || 1;
>
> Does this want to be // rather than ||? You otherwise cannot set no_ack
> to 0.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bobtfish/AnyEvent-RabbitMQ/pull/15/files#r26893983>.
>